Chris-Denmark/nss-aquarium
# javascript work to automate html

## Changes made:
1) automated the fish, quote, location, and tips sections using javascript so they no longer need to be hard coded using html.

## Steps to check:
1) proceed to any ___DataProvider file.
2) add another object to the array that includes information in a similar format to the other objects in the array.
3) run the serve command.
4) proceed to your web browser of choice and paste the serve url.
5) observe the section you added an item to.
